src/services/Solana/GetData/GetCommunityData.tsx

```diff
@@ -22,9 +23,9 @@ export async function GetFungibleCommunityData(
   tokenRegistry: TokenInfoMap,
   token: IfungibleToken
 ): Promise<IfungibleTokenCommunity> {
-  let communityData;
```

Comment:
As it is almost certain that we will store more data on the server than categories, I prefer that we keep using the communityData abstraction as a way to encapsulate all the community data stored on our server.

Reply:
not for now, when we ll have we wil do

Reply:
Again, it's better to have an abstraction rather than literal categories as we will certainly need to include more data. But it's ok, if you don't feel comfortable doing that now we can change later.

Reply:
yeah it was easier to do it like that, we only have categories for now and it will be subject to a lot of changes soon so it ll be better to keep it simple for now at least